package.json

```diff
@@ -18,17 +18,19 @@
     "build": "npm run bundle && npm run minify",
     "bundle": "rollup -i src/index.js -o dist/logger.js -m -f umd -n logger",
     "minify": "uglifyjs dist/logger.js -o dist/logger.js --mangle --compress warnings=false -p relative --source-map dist/logger.js.map",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "format": "prettier --semi false --write '{src,test}/**/*.js'"
   },
   "babel": {
     "presets": "es2015"
   },
   "devDependencies": {
     "babel-preset-es2015": "^6.24.1",
+    "hyperapp": "^0.12.0",
```

Comment:
@okwolf Wasn't it going to be a peer-dep? 🤔

Reply:
@JorgeBucaran it's both if you use it in tests and also want consumers of your library to install it: https://github.com/hyperapp/logger/pull/4#discussion_r137981399

Reply:
@okwolf Ah, that makes sense! Thanks.